packages/@sanity/default-layout/src/navbar/presence/PresenceMenuItem.tsx

```diff
@@ -0,0 +1,62 @@
+import React, {forwardRef, useMemo} from 'react'
+import {GlobalPresence} from '@sanity/base/presence'
+import {orderBy} from 'lodash'
+import * as PathUtils from '@sanity/util/paths'
+import {Card, MenuItem} from '@sanity/ui'
+import {UserAvatar} from '@sanity/base/components'
+import {LinkIcon} from '@sanity/icons'
+import {IntentLink} from '@sanity/state-router/components'
```

Comment:
Can this be imported from `@sanity/base/router`?

Reply:
Yes, updating the import 👍